plugins/modules/aci_interface_policy_storm_control.py

```diff
@@ -0,0 +1,434 @@
+        ),
+        dict(
+            config_input=module.params.get("multicast_configuration"),
+            rates=dict(rate=dict(percentage="mcRate", pps="mcRatePps"), burst_rate=dict(percentage="mcBurstRate", pps="mcBurstPps")),
+        ),
+        dict(
+            config_input=module.params.get("unicast_configuration"),
+            rates=dict(rate=dict(percentage="uucRate", pps="uucRatePps"), burst_rate=dict(percentage="uucBurstRate", pps="uucBurstPps")),
+        ),
+    ]
+
+    for config in stom_control_types_configs:
+        config_input = config.get("config_input")
+        rates = config.get("rates")
+        if config_input is not None:
+            if config_input.get("rate_type") == "percentage":
+                for rates_type, rates_attributes in rates.items():
+                    input = config_input.get(rates_type)
+                    if input is not None and not (0 <= float(input) <= 100):
+                        module.fail_json(
+                            msg="If argument rate_type is percentage, argument {0} needs to be a value between 0 and 100 inclusive, got {1}".format(
+                                rates_type,
+                                input,
+                            )
+                        )
+                    else:
+                        rates_input[rates_attributes.get("percentage")] = "{0:.6f}".format(float(input))
+                        rates_input[rates_attributes.get("pps")] = "unspecified"
+            elif config_input.get("rate_type") == "pps":
+                for rates_type, rates_attributes in rates.items():
+                    rates_input[rates_attributes.get("percentage")] = None
+                    rates_input[rates_attributes.get("pps")] = config_input.get(rates_type)
+        else:
+            for rates_type, rates_attributes in rates.items():
+                rates_input[rates_attributes.get("percentage")] = None
+                rates_input[rates_attributes.get("pps")] = None
+
+    aci.construct_url(
+        root_class=dict(
+            aci_class="infraInfra",
+            aci_rn="infra",
+        ),
+        subclass_1=dict(
+            aci_class="stormctrlIfPol",
+            aci_rn="stormctrlifp-{0}".format(storm_control_policy),
+            module_object=storm_control_policy,
+            target_filter={"name": storm_control_policy},
+        ),
+    )
+
+    aci.get_existing()
+
+    if state == "present":
+        aci.payload(
+            aci_class="stormctrlIfPol",
+            class_config=dict(
+                name=storm_control_policy,
+                descr=description,
+                nameAlias=name_alias,
+                isUcMcBcStormPktCfgValid=storm_control_types,
+                stormCtrlAction=storm_control_action,
+                stormCtrlSoakInstCount=storm_control_soak_action,
+                **rates_input
```

Comment:
Do we want to do ** since it is not supported in python below 3.4, thus also no 2.7. We could leverage update instead by defining the dict before and calling update before passing into payload function.

Another option if we do python 3.9 and up is the | operator ( if not mistaken about the version )